components/blitz/src/ome/services/blitz/measurements/Helper.java

```diff
@@ -31,8 +29,8 @@
      * Factory method to generate Helper instances from the LSID of a given
      * column.
      * 
-     * @param lsid
-     * @return
+     * @param lsid The lsid
+     * @return See above.
```

Comment:
These were removed elsewhere. Was there a guideline discussed before the cleaning started?